app/src/main/java/fr/free/nrw/commons/nearby/Place.java

```diff
@@ -54,6 +58,9 @@ public Place(Parcel in) {
         this.pic = (picString == null) ? "":picString;
         String destroyedString = in.readString();
         this.destroyed = (destroyedString == null) ? "":destroyedString;
+        String endTimeString = in.readString();
+        this.endTime = (endTimeString == null) ? "":endTimeString;
+        this.exists = (destroyedString == null) && (endTimeString == null);
```

Comment:
please add a comment next to this && operation where you briefly explained what is happening